fluent-bit/deb/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Internal variables
+PELION_PACKAGE_NAME="fluent-bit"
+PELION_PACKAGE_DIR=$(cd "`dirname \"$0\"`" && pwd)
+
+PELION_PACKAGE_SOURCE_PREPARATION_CALLBACK=pelion_fluent_bit_source_preparation_cb
+
+declare -A PELION_PACKAGE_COMPONENTS=(
+    ["https://github.com/fluent/fluent-bit.git"]="v1.7.9")
+source "$PELION_PACKAGE_DIR"/../../build-env/inc/build-common.sh
+
+function pelion_fluent_bit_source_preparation_cb() {
+    cd "$PELION_SOURCE_DIR/$PELION_PACKAGE_NAME/$PELION_PACKAGE_NAME"
+    rm -rf debian
```

Comment:
Can you add a comment here about why we do this?  It seems really out of place.

Reply:
👍